go/gpg/main.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"encoding/base64"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/ProtonMail/go-crypto/openpgp"
+	"github.com/ProtonMail/go-crypto/openpgp/packet"
+	"github.com/google/go-github/v54/github"
+	"golang.org/x/crypto/nacl/box"
+)
+
+var (
+	org        = flag.String("org", "", "GitHub organization name")
+	secretName = flag.String("secret", "GPG_PRIVATE_KEY", "GitHub organization secret name")
+	visibility = flag.String("visibility", "all", "GitHub organization secret visibility")
+
+	gpgName    = flag.String("gpg-name", "opentofu", "GPG name")
+	gpgComment = flag.String("gpg-comment", "This is the key used to sign opentofu providers", "GPG comment")
+	gpgEmail   = flag.String("gpg-email", "your.email@example.com", "GPG comment")
+
+	pat string
+)
+
+func main() {
+	pat = os.Getenv("GITHUB_PAT")
+	if pat == "" {
+		panic("GITHUB_PAT environment variable not set")
+	}
+
+	flag.Parse()
+
+	config := &packet.Config{
+		DefaultCipher:          packet.CipherAES256,
+		DefaultCompressionAlgo: packet.CompressionZLIB,
+		CompressionConfig: &packet.CompressionConfig{
+			Level: packet.BestCompression,
+		},
+		RSABits: 4096,
+	}
+
+	es, err := openpgp.NewEntity(*gpgName, *gpgComment, *gpgEmail, config)
+	if err != nil {
+		fmt.Println(err)
+		return
```

Comment:
Why not just fatal like in other places?

Reply:
My bad! Should totally be just that.